docs/root/configuration/listeners/network_filters/kafka_broker_filter.rst

```diff
@@ -0,0 +1,86 @@
+.. _config_network_filters_kafka_broker:
+
+Kafka Broker filter
+===================
+
+The Apache Kafka broker filter decodes the client protocol for
+`Apache Kafka <https://kafka.apache.org/>`_, both the requests and responses in the payload.
+The message versions in `Kafka 2.0 <http://kafka.apache.org/20/protocol.html#protocol_api_keys>`_
+are supported.
+The filter attempts not to influence the communication between client and brokers, so the messages
+that could not be decoded (due to Kafka client or broker running a newer version than supported by
+this filter) are forwarded as-is.
```

Comment:
Can you link to and discuss the relationship with the other Kafka filter?

Reply:
The one in this PR is the first Kafka filter (what's present in `master` branch is only protocol codec code).